doc/modules/cross_validation.rst

```diff
@@ -290,6 +290,40 @@ both testing and training. Notice that the folds do not have exactly the same
 size due to the imbalance in the data.
 
 
+.. _homogeneous_time_series_cv:
+
+HomogeneousTimeSeriesCV
+-----------------------
+
+:class:`HomogeneousTimeSeriesCV` is a variation of *k-fold* which 
+returns first :math:`k` folds as train set and the :math:`(k+1)` th 
+fold as test set. Note that unlike standard cross-validation methods, 
+successive training sets are supersets of those that come before them.
+Also, its foldsizes is not in accordance with :class:`KFold`,
+it add all surplus data to the first training partition, which
+is always used to train the model.
+
+This class can be used to cross-validate homogeneous time series data,
+i.e., samples are observed at fixed time intervals.
```

Comment:
thanks! 

I've addressed the comments from you and other reviewers, have a look once you get time!